website/versioned_docs/version-v1.x/contracts.md

```diff
@@ -31,8 +31,6 @@ constructor(
     signUpGatekeeper = _signUpGatekeeper;
     initialVoiceCreditProxy = _initialVoiceCreditProxy;
 
-    signUpTimestamp = block.timestamp;
```

Comment:
Is it supposed to be removed?

Reply:
yes I removed it from the code as well as it wasn't used anywhere. Let me know if you think we should have kept it in the code